cabal-install/Distribution/Client/Dependency/Types.hs

```diff
@@ -1,16 +1,4 @@
 {-# LANGUAGE DeriveGeneric #-}
------------------------------------------------------------------------------
--- |
--- Module      :  Distribution.Client.Dependency.Types
--- Copyright   :  (c) Duncan Coutts 2008
--- License     :  BSD-like
---
--- Maintainer  :  cabal-devel@haskell.org
--- Stability   :  provisional
--- Portability :  portable
---
--- Common types for dependency resolution.
```

Comment:
I'm okay with removing the header, but it'd be nice to preserve the top comment about the purpose of the module.